packages/alfa-aria/src/node.ts

```diff
@@ -365,7 +365,10 @@ export namespace Node {
         // Get the children implicitly owned by the element. These are the
         // children in the flat tree that are neither claimed already nor
         // explicitly owned by the element.
-        const implicit = node
+
+        // children of iframe are not included, they are fallback content for legacy browsers…
+        // @see https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element
+        const implicit = node.name === "iframe" ? Sequence.empty<dom.Node>() : node
```

Comment:
This only causes the implicit children to be excluded, not the explicit children.

Reply:
That was intentional as I assumed that `aria-own` should be respected, but I was indeed wrong…
```html
<iframe><span id="foo">Foo</span></iframe>
<div aria-label="foofoo" aria-owns="foo">Owns foo</div>
```
The `<div>` doesn't actually own `foo`…